dashboard/src/main/java/com/github/ladicek/rourka/jenkins/JenkinsProperties.java

```diff
@@ -0,0 +1,29 @@
+package com.github.ladicek.rourka.jenkins;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class JenkinsProperties {
+
+    private static Properties properties;
+
+    /*
+     Open and load a properties file
+     */
+    static {
+        try {
+            ClassLoader loader = JenkinsProperties.class.getClassLoader();
```

Comment:
This won't work. I will want to store the credentials into an OpenShift secret, which will be exposed either as environment variables or a file. Probably a file.